src/ladds/particle/Constellation.h

```diff
@@ -167,4 +211,9 @@ class Constellation {
    * altitude. uses altitudeDeviation as parameter
    */
   std::normal_distribution<double> distribution;
+
+  /**
+   * variable used to give every satellite that is part of a constellation an id
+   */
+  static size_t idBase;
```

Comment:
I would say then it would be best to just distribute the IDs at initialization: Tracked objects: [0 ... N-1], Constellation satellites: [N ... M-1] and then debris from breakups start from M. Because if we make the ID dependent on the time of insertion, then particles of the same constellation are not neighboring anymore and statistics looking at individual constellations based on the ID wouldn't be possible. Should I do it like this, or as you said?

Reply:
yep sounds reasonable. Implement it like you suggested!